migrator/data/DataExporter.java

```diff
@@ -49,7 +49,7 @@
 public class DataExporter {
     private static final Logger LOG = LoggerFactory.getLogger(DataExporter.class);
 
-    private final TypeDB typedb;
+    private final TypeDB.DatabaseManager databaseMgr;
```

Comment:
Now that `RocksTypeDB` has been deleted, the reference is updated to point to `databaseManager`.